### PR TITLE
feat: remove `find_ouptuts` method

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

Removes the `find_outputs` from `Client`

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes #issue_number`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: .changes/client-find-outputs-remove.md
Comment (thibault-martinez) [inline .changes/client-find-outputs-remove.md:1]: No .change if you're not touching the old bindings

File: bindings/nodejs/CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `CommonOutput::{setNativeTokens, setFeatures}`;
 - `ImmutableFeaturesOutput::setImmutableFeatures`;
 - `StateMetadataOutput::setStateMetadata`;
+- `Client::find_outputs()` method;
```
Comment (Thoralf-M) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
- `Client::findOutputs()` method;
```

File: .changes/client-find-outputs-remove.md
Comment (qrayven) [reply .changes/client-find-outputs-remove.md:1]: done

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/CHANGELOG.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Prefix-hex string values replaced with boxed slices in Dtos;
 - `MilestoneEssence::new` now takes generic metadata for convenience;
 - `ParametersMilestoneOption::new` now accepts boxed slice;
-- `{SecretManage, ClientBlockBuilder}::sign_transaction` return type from `Payload` to `TransactionPayload`;
```
Comment (thibault-martinez) [inline sdk/CHANGELOG.md]: ?

File: sdk/CHANGELOG.md [hunk 3]
```diff
@@ -34,13 +34,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Prefix-hex string values replaced with boxed slices in Dtos;
 - `MilestoneEssence::new` now takes generic metadata for convenience;
 - `ParametersMilestoneOption::new` now accepts boxed slice;
```
Comment (Thoralf-M) [inline sdk/CHANGELOG.md]:
```suggestion
- `ParametersMilestoneOption::new` now accepts boxed slice;
- `{SecretManage, ClientBlockBuilder}::sign_transaction` return type from `Payload` to `TransactionPayload`;
```

File: sdk/CHANGELOG.md (hunk 2 above)
Comment (qrayven) [reply sdk/CHANGELOG.md]: I don't know how It happened

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**